.travis.yml

```diff
@@ -8,6 +8,7 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then tools/apt-get-install-deps.sh; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then tools/apt-get-install-qemu-arm.sh; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then tools/brew-install-deps.sh; fi
+  - if [[ "$TARGET" == "build-zephyr" ]]; then tools/apt-get-install-zephyr-deps.sh; fi
```

Comment:
The 3rd build fails (the one with toolchain) if I choose TRAVIS_OS_NAME == linux. I think there are some conflicts in the packages installed.

Travis error is:
CMake Error at CMakeLists.txt:16 (project):
  The CMAKE_C_COMPILER:
    arm-linux-gnueabihf-gcc
  is not a full path and was not found in the PATH.
  Tell CMake where to find the compiler by setting either the environment
  variable "CC" or the CMake cache entry CMAKE_C_COMPILER to the full path to
  the compiler, or to the compiler name if it is in the PATH.

Maybe I can change the order of the dependencies scripts...so that I call the install-zephyr.sh before all of them, perhaps this could fix the conflict....Although I thought it was better for that dependency script to be called only for the build in which it's actually building and running the zephyr with jerryscript.